### DISCUSSION TITLE
how to make web page from file .html and file .css save in SPIFFS

### DISCUSSION THREAD

**Discussion by hongthiet1994** [Hieromon/AutoConnect#513] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29925688/181675142-28bd558e-b559-4602-959b-7fa1c18e3249.png)

I have 2 files saved into spiffs (index.html and style.css)

i want to make page like : 192.168.4.2/config

**Comment by Hieromon** [comment, marked as answer]
@hongthiet1994
What do you mean? AutoConnect does not handle native HTML (which of course may include CSS). AutoConnect is not a server. It is a WiFi connection utility with a facility that refactors HTTP request handlers as [custom web pages](https://hieromon.github.io/AutoConnect/achandling.html#custom-web-page-handler-programming-model).

A simple static server that responds to HTML content stored in the file system should use natively the WebServer library. Suitable examples for this are provided as _SDWebServer_ or _WebServer_ for each of Espressif's Arduino cores:
https://github.com/espressif/arduino-esp32/blob/master/libraries/WebServer/examples/SDWebServer/SDWebServer.ino
https://github.com/esp8266/Arduino/blob/master/libraries/ESP8266WebServer/examples/WebServer/WebServer.ino
They are all skeletons sufficient for a static HTTP server.
Also, the [documentation](https://github.com/esp8266/Arduino/tree/master/libraries/ESP8266WebServer/examples/WebServer) that comes with the [WebServer library](https://github.com/esp8266/Arduino/tree/master/libraries/ESP8266WebServer) for the ESP8266 Arduino core also explains how to implement an HTTP server. Once you have read through their documentation, examine what implementation models are feasible for your requirements.

If you already know the [AutoConnect programming model](https://hieromon.github.io/AutoConnect/achandling.html#custom-web-page-handler-programming-model) and all you want to do is [save a JSON description](https://hieromon.github.io/AutoConnect/achandling.html#loading-saving-autoconnectelements-with-json) of your custom web page to SPIFFS, AutoConnect has provided an example for that as [mqttRSSI_FS](https://github.com/Hieromon/AutoConnect/tree/master/examples/mqttRSSI_FS).
